docs/abssys.rst

```diff
@@ -0,0 +1,65 @@
+.. highlight:: rest
+
+******************
+AbsSystem Class
+******************
+
+.. index:: AbsSystem
+
+Overview
+========
+
+This Class is designed to organize and analyze an
+absorption system.  This is generally constructed
+of one or more (:ref:`AbsComponent Class`).
+The base class is abstract, i.e. one must instantiate
+one of its flavors (e.g. HILyman, MgII, LLS, DLA).
```

Comment:
I'm not convinced about the flavors. I think I'd prefer to have only a generic AbsSystem class, and depending on what its constituents components are, one can call them one way or another. In this way one can eventually combine HI components with MgII components in the same absorption system.